soh/soh/ResourceManagerHelpers.cpp

```diff
@@ -11,9 +11,18 @@
 #include "resource/type/Array.h"
 #include "resource/type/Skeleton.h"
 #include "resource/type/PlayerAnimation.h"
+#include "soh/ActorDB.h"
+#include "soh/resource/type/Scene.h"
+#include "soh/resource/type/scenecommand/SetRoomList.h"
+#include "soh/resource/type/scenecommand/SetActorList.h"
+#include "soh/resource/type/scenecommand/SetObjectList.h"
 #include <Fast3D/gfx_pc.h>
 #include <DisplayList.h>
 
+bool delayedAltAssetLoad = false;
+s16 unloadScene = -1;
+std::shared_ptr<BS::thread_pool> rmhThreadPool = std::make_shared<BS::thread_pool>(1);
```

Comment:
what does `rmh` mean? resource manager hooks?

i know this is a bit of a nit but this name feels less than ideal to me. someone unfamiliar would have to guess what `rmh` means, but the variable name spells out `ThreadPool` (which can be inferred from the type)

i don't have a perfect suggestion for a name but something like `resourceThreads` might work? i'd love to hear any ideas others have

Reply:
I'm down for changing it. It was named for the fie it's in, ResourceManagerHelpers.